### PR TITLE
Fix simple install

### PR DESCRIPTION
**Pull request by justin808**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,3 +139,10 @@ you may have issues.
 * [boostrap-webpack](https://github.com/bline/bootstrap-webpack).
 * DylanLukes/bootstrap-sass-webpack
 
+# Known Issues
+1. Automatic Dependency loading is currently problematic. If you "touch" the either of the customization files listed in
```
Comment (mbreining) [inline README.md]: If you "touch" -either- of the ...

File: README.md [hunk 2]
```diff
@@ -109,6 +110,7 @@ module.exports = {
       // Needed for the css-loader when [bootstrap-webpack](https://github.com/bline/bootstrap-webpack)
       // loads bootstrap's css.
       { test: /\.woff(\?v=\d+\.\d+\.\d+)?$/,   loader: "url?limit=10000&minetype=application/font-woff" },
```
Comment (mbreining) [inline README.md]: I guess you could also do:

`{ test: /\.woff2?(\?v=\d+\.\d+\.\d+)?$/,   loader: "url?limit=10000&minetype=application/font-woff" },`

**Comment by mbreining** [comment]
Looks good to me.

File: README.md (hunk 2 above)
Comment (justin808) [inline README.md]: I thought about that, but I left it explicit. I'm not sure what the rest of the regexp does.

**Pull request merged**